scripts/organisational-role-assignment.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+## Usage: ./organisational-role-assignment.sh [username] [password] [microservice_name]
+##
+## Options:
+##    - username: Role assigned to user in generated token. Default to `ccd-import@fake.hmcts.net`.
+##    - password: Password for user. Default to `London01`. 
+##    - microservice_name: Name of the microservice. Default to `ccd_gw`.
+##
+## Returns a valid IDAM service token for the given microservice.
+
+USERNAME=${1:-ccd-import@fake.hmcts.net}
+PASSWORD=${2:-London01}
+MICROSERVICE="${3:-ccd_gw}"
```

Comment:
is the microservice used anywhere ?

Reply:
Ah thats a bug should have been passed to the idam0service-token.sh script